core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -824,11 +827,11 @@ public static Jenkins getActiveInstance() throws IllegalStateException {
      * {@link #get} is what you normally want.
      * <p>In certain rare cases you may have code that is intended to run before Jenkins starts or while Jenkins is being shut down.
      * For those rare cases use this method.
-     * <p>In other cases you may have code that might end up running on a remote JVM and not on the Jenkins master.
+     * <p>In other cases you may have code that might end up running on a remote JVM and not on the Jenkins controller or built-in node.
      * For those cases you really should rewrite your code so that when the {@link Callable} is sent over the remoting channel
```

Comment:
```suggestion
     * For those cases, you really should rewrite your code so that, when the {@link Callable} is sent over the remoting channel,
```